src/site/stages/build/drupal/graphql/GetAllPages.graphql.js

```diff
@@ -57,5 +59,6 @@ module.exports = `
     ${sidebarQuery}
     ${facilitySidebarQuery}
     ${alertsQuery}
+    ${outreachAssetsQuery}
```

Comment:
Should this also be registered to `GetLatestPageById`?

Reply:
No, I don't think so. Since this is processed as a separate file, the preview server won't ever use the result.